packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/shippingMethod.js

```diff
@@ -99,6 +69,53 @@ const ShippingMethod = props => {
                 />
             </Fragment>
         );
+    } else {
+        // We're either initializing or editing.
+        const lowestCostShippingMethodSerializedValue = shippingMethods.length
+            ? shippingMethods[0].serializedValue
+            : '';
+        const lowestCostShippingMethod = {
+            shipping_method: lowestCostShippingMethodSerializedValue
+        };
+
+        const isContinueDisabled = pageIsUpdating;
+        const maybeContinueButton = !isLoading ? (
+            <div className={classes.formButtons}>
+                <Button
+                    priority="normal"
+                    type="submit"
+                    disabled={isContinueDisabled}
+                >
+                    {'Continue to Payment Information'}
+                </Button>
+            </div>
+        ) : null;
+
+        const editingContents = (
+            <Form
+                className={classes.form}
+                initialValues={lowestCostShippingMethod}
+                onSubmit={handleSubmit}
+            >
+                <ShippingRadios
+                    isLoading={isLoading}
+                    shippingMethods={shippingMethods}
+                />
+                {maybeContinueButton}
+            </Form>
+        );
+
+        const bodyContents =
+            displayState === displayStates.INITIALIZING
+                ? initializingContents
+                : editingContents;
```

Comment:
The only logic that actually changed is the addition of what to do when `displayState` is `INITIALIZING`. The other changes here are just for readability.